build.gradle

```diff
@@ -21,11 +21,12 @@ buildscript {
     ext.kotlin_version = '1.4.0'
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        gradlePluginPortal()
```

Comment:
Why do we need this? Are there plugins not resolving?

Reply:
It's just for https://github.com/ben-manes/gradle-versions-plugin